apps/sms/index.html

```diff
@@ -115,10 +115,10 @@
     <script defer src="views/conversation/js/subject_composer.js"></script>
     <script defer src="views/conversation/js/compose.js"></script>
     <script defer src="views/shared/js/waiting_screen.js"></script>
-    <script defer src="views/shared/js/activity_picker.js"></script>
```

Comment:
@gaye , as you could see I moved activity_picker.js from views/shared/js to shared/js/links. And I have used this module in sms and calendar apps. I tried to replace all links that necessary in sms app and after testing the app, as I could notice, it works as it worked.
But would you do me a favor and invite some competent reviewer for the sms app to review my code from this side?  

Other your comments are fixed.

Thanks,